src/api/http.rs

```diff
@@ -219,17 +210,29 @@ fn partition_by_id(
         })
 }
 
-#[put("/vdisks/<vdisk_id>/partitions/<partition_id>/<action>")]
+#[get("/vdisks/<vdisk_id>/partitions/<partition_id>/<action>")]
```

Comment:
any change should not be GET it should be `POST`  or `PUT`  or `PATCH`. Here lets use `PUT` as we replacing state

Reply:
Let make it `POST`)

Reply:
Fixed